Acornfile

```diff
@@ -5,100 +5,127 @@ info:        localData.info
 icon:        "./icon.png"
 
 args: {
-  // Name of the default database
-  dbName: "mydb"
+	// Name of the default database
+	dbName: "mydb"
 
-  // Name of the default user
-  dbUser: ""
+	// Name of the default user
+	dbUser: ""
+
+	// Number of MongoDB instances to run in the MongoDB replica set
+	replicas: number > 0 || default 1
 }
 
 services: db: {
 	default:   true
-	container: "mongodb"
+	container: "mongodb-0"
```

Comment:
Since `containerLabels` are disabled in manager for security reasons, we'll need support for setting multiple containers by name before `args.replicas > 1` will work properly.

At this point, I'm also not sure if exposing a mongodb replica set via a service LB will work properly. I've seen a lot of examples of clients consuming connection strings that contain all members.

Reply:
see https://github.com/acorn-io/runtime/issues/2327